Scripts/CommonServerPython/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+Added cveRegex to validate domainGlob format.
```

Comment:
`cveRegex`?? It looks like you made a mistake here.

Reply:
Also, please change it to `DomainGlob`.

Reply:
This is outdated